core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -463,8 +463,11 @@ public function compare($blnReturnBuffer=false)
 				$arrOrderFields = $objDcaExtractor->getOrderFields();
 
 				// Find the changed fields and highlight the changes
-				foreach ($to as $k=>$v)
+				foreach (array_keys(array_merge($to, $from)) as $k)
 				{
+					$to[$k] = $to[$k] ?? null;
+					$from[$k] = $from[$k] ?? null;
```

Comment:
```suggestion
					$to[$k] ??= null;
					$from[$k] ??= null;
```

Since we are on PHP 7.4+, we should be able to use the null coalesce assignment operator, shouldn‘t we?